docs/source/demos/local_fleet_demo.rst

```diff
@@ -33,7 +33,7 @@ The first step is to launch the first Ursula on the network by running:
 
 .. code::
 
-   $ python run_lonely_demo_ursula.py
+   $ python run_lonely_ursula.py
```

Comment:
Do we want to provide the full path i.e. `python scripts/local_fleet/run_lonely_ursula.py` (Same for other files in commands provided below)? Or at least indicate in a previous paragraph where these files are located - then we don't have to change each command to use a relative path?